pkg/utils/helper.go

```diff
@@ -44,10 +44,10 @@ func FormatSize(size int64) string {
 	return fmt.Sprintf("%.2fMiB", mbSize)
 }
 
+// ValidateUserName checks if the username is valid by length and allowed characters.
 func ValidateUserName(username string) bool {
-	pattern := `^[a-zA-Z0-9]{1,255}$`
-	re := regexp.MustCompile(pattern)
-	return re.MatchString(username)
+	username = strings.TrimSpace(username)
```

Comment:
I believe this code may not work as expected. If you enter `harbor-cli` with extra spaces, the `strings.TrimSpace(username)` doesn't properly store the value in the `login.LoginView` variable. Could you please fix this? @Roaster05